package.json

```diff
@@ -39,14 +39,15 @@
     "eslint-config-stripes": "^0.0.1",
     "mocha": "^3.1.2",
     "mock-require": "^2.0.1",
-    "react-addons-test-utils": "^15.3.2"
+    "react-addons-test-utils": "^15.3.2",
+    "serialize-javascript": "^1.4.0",
+    "webpack-virtual-modules": "^0.1.8"
   },
   "dependencies": {
     "@folio/stripes-components": "^1.6.0",
-    "@folio/stripes-connect": "^2.7.0",
-    "@folio/stripes-loader": "^0.0.2",
+    "@folio/stripes-connect": "folio-org/stripes-connect#3c2714a189c2296f389fc6ee4e79a7ee7c962949",
     "@folio/stripes-logger": "^0.0.2",
-    "@folio/stripes-redux": "^2.0.0",
+    "@folio/stripes-redux": "folio-org/stripes-redux#138803e34c8472b657ef62df76316d941177f75f",
```

Comment:
Could we just cut new releases of `stripes-redux` and `stripes-connect`? That seems more reliable than a commit hash.

Reply:
Yes. Until those releases are cut, things are broken for those working against the head of stripes-core. These hashes would be removed prior to the next release.